Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/CvarAdapter.cpp

```diff
@@ -142,7 +197,7 @@ namespace AZ::DocumentPropertyEditor
         bool BuildEditorForCvar(AdapterBuilder& builder, ConsoleFunctorBase* functor)
         {
             // Check all possible type IDs for our supported types and try to create a property editor
-            return TryBuildNumericEditor<int8_t, int16_t, int32_t, int64_t, uint8_t, uint16_t, uint32_t, uint64_t, float, double>(
+            return TryBuildNumericEditor<AZ::s8, AZ::u8, AZ::s16, AZ::u16, AZ::s32, AZ::u32, AZ::s64, AZ::u64, int64_t, size_t, float, double>(
```

Comment:
`int64_t` and `size_t` will not compile on Linux.
This should be `long` and `unsigned long`  as `AZ::s64` is always `long long` and `AZ::u64` is always `unsigned long long`.

Reply:
fixed, thanks!